src/components/Patient/PatientHome.tsx

```diff
@@ -236,181 +240,162 @@ export const PatientHome = (props: {
+                                <li key={skill.skill_object.id}>
+                                  {skill.skill_object.name}
+                                </li>
+                              ))}
+                            </ul>
+                          )}
                         </span>
-                      </span>
-                    )}
-                  {patientData.is_vaccinated && (
-                    <Chip
-                      variant="custom"
-                      size="small"
-                      className="bg-blue-100 text-blue-800"
-                      startIcon="l-syringe"
-                      text={t("vaccinated")}
-                    />
-                  )}
-                  {patientData.allow_transfer ? (
-                    <Chip
-                      variant="warning"
-                      size="small"
-                      startIcon="l-unlock"
-                      text={t("transfer_allowed")}
-                    />
-                  ) : (
-                    <Chip
-                      startIcon="l-lock"
-                      size="small"
-                      text={t("transfer_blocked")}
-                    />
+                      </div>
+                    </div>
                   )}
-
-                  {patientData.gender === 2 && (
-                    <>
-                      {patientData.is_antenatal &&
-                        isAntenatal(
-                          patientData.last_menstruation_start_date,
-                        ) && (
-                          <Chip
-                            variant="custom"
-                            size="small"
-                            className="border-pink-300 bg-pink-100 text-pink-600"
-                            startIcon="l-baby-carriage"
-                            text={t("antenatal")}
-                          />
+                  {patientData?.last_consultation?.assigned_to_object && (
+                    <div>
+                      <p className="text-xs font-normal leading-tight text-gray-600">
+                        {t("assigned_doctor")}:
+                      </p>
+                      <div className="mt-1 flex space-x-2 text-sm font-semibold leading-tight text-gray-900">
+                        <p>
+                          {formatName(
+                            patientData.last_consultation.assigned_to_object,
+                          )}
+                        </p>
+                        {patientData?.last_consultation?.assigned_to_object
+                          .alt_phone_number && (
+                          <a
+                            href={`https://wa.me/${patientData.last_consultation.assigned_to_object.alt_phone_number.replace(/\D+/g, "")}`}
+                            target="_blank"
+                            rel="noreferrer"
+                            className="flex items-center space-x-1 text-xs text-green-500"
```

Comment:
missed this one out? switch to primary.

Reply:
yep

Reply:
Also switched to secondary for gray

Reply:
> Also switched to secondary for gray

But text-gray-900 cant be switched as this offers dark colour for text which is not provided by secondary